app/models/certificate/docker_registry.rb

```diff
@@ -0,0 +1,2 @@
+class Certificate::DockerRegistry < ActiveRecord::Base
+end
```

Comment:
IMO it would be better if we only had `Certificate` model with polymorphic associations (http://guides.rubyonrails.org/association_basics.html#polymorphic-associations), so we could reuse certificates among different future services (only registries for now).

If we add other models that will require certificates, the customer will have to duplicate certificates for each one if we don't use the polymorphic association.

Reply:
done